code/controllers/subsystem/vote.dm

```diff
@@ -213,6 +230,8 @@ SUBSYSTEM_DEF(vote)
 					choices[i] = 0
 			if("roundtype") //CIT CHANGE - adds the roundstart secret/extended vote
 				choices.Add("secret", "extended")
+			if("dynamic")
+				choices.Add("extended",PEACE2,PEACE1,BALANCED,CHAOS1,CHAOS2)

```

Comment:
You could as well add a "secret" option here too, but I doubt both are likely to be ever choosen since they still require over fifty percent of all votes to trigger.

Reply:
Yeah, that's a worry--could make it so that a sufficiently low mean makes it do dynamic_classic_secret.

Reply:
Upon really checking the code in the process of #9582, looks like secret's basically irrelevant as long as threat level's low, and I can't really think of a way to take those votes into account for if dynamic ends up winning anyway (-1? -2 like extended?), so I'll hold off on this until testing.